eval/corpus/yaml/ha_automation.yaml

```
@@ -0,0 +1,188 @@
+      - service: alarm_control_panel.alarm_arm_away
+        target:
+          entity_id: alarm_control_panel.home
+      - service: light.turn_off
+        target:
+          entity_id: all
+      - service: climate.set_temperature
+        target:
+          entity_id: climate.main_floor
+        data:
+          temperature: 16
+    action:
```

YAMLlint (1.38.0)

[error] duplication of key "action" in mapping

(key-duplicates)